asllib/libdir/stdlib.asl

```diff
@@ -46,6 +51,30 @@ begin
     return (a MOD 2) == 1;
 end;
 
+func AlignUpP2(x: integer, y: integer) => integer
```

Comment:
We probably need a line of comment per function. Ideally with the precondition/assertion on the arguments (e.g., non-negative).